### DISCUSSION TITLE
How do I change the AI voice in the callback and voicemail

### DISCUSSION THREAD

**Discussion by jvidamins** [twilio-professional-services/flex-project-template#679] [Q&A]
Anyone know? In the regular install of Flex, after I deployed their version of callback and voicemail, I just hit settings right inside Flex, change the AI voice to whatever I want, then click save. I don't see a way to do that in this version. I found where to changed it in the service function code, but it won't let me make changes since it's uneditable from Github.

**Comment by jvidamins** [comment]
I also desperately need to change the hold music, again, which was super easy in the default Flex install version.

**Comment by dremin** [comment]
To change any of these options in the in-queue callback experience, you will need to edit `serverless-functions/src/functions/features/callback-and-voicemail/studio/wait-experience.protected.js`. The settings are near the top of the file within the `options` object.

> it won't let me make changes since it's uneditable from Github.

You should be able to edit the files in your own repo without issue, could you please provide more detail as to what you're seeing?
